### PR TITLE
Updating Figures and Text - removing Interworking

### PR DESCRIPTION
**Pull request by dfedyk**
Work In progress - Wanted to get the figure changes out for review.   Need to check if the overly wording is complete.  There was a lot of inter working text - I have commented out.

### CODE REVIEW

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 1]
```diff
@@ -164,45 +165,48 @@
...
+       :  Link  :     /  ,-----.  \   :  Link  :   /  ,-----. \
+       +........+     +-[  Sub  ]-+   +........+   +-[  TSN  ]-+
                         [Network]                    [Network]
                          `-----'                      `-----'
 
-        |<- TSN ->| |<------ DetNet MPLS ------>| |<-- TSN --->|
+                    |<------ DetNet MPLS ------>|
+        |<---------------------- TSN   --------------------->|
+
 ]]></artwork>
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml:203]: image

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 2]
```diff
@@ -164,45 +165,48 @@
...
-	"associates" the DetNet flows and native flows (i.e., TSN Streams)
-	at the edge of the DetNet domain. This ensures that the DN Flow is 
-	properly served at the Edge node (and inside the domain).
+        In this example, edge nodes provide a service proxy function that 
+        "associates" the DetNet flows and native flows (i.e., TSN Streams) at
+        the edge of the DetNet domain.  The TSN streams are treated as DetNet
+        Applicaions.  The service proxy behaves as a TSN relay node as far as
+        the TSN flows are concerned. For DetNet MPLS the service proxy presents
+        the TSN services as an application flow. 
   </t>
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml:213]: Not fully happy with the changes. I would prefer something like this:
"TSN streams are treated as App-flows for DetNet. The whole DetNet domain behaves as a TSN relay node for the TSN streams. The service proxy behaves as a port of that TSN relay node. "
Also concerns on this sentence: "For DetNet MPLS the service proxy presents the TSN services as an application flow." It is more the TSN stream, not the TSN service. I would delete this sentence.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 3]
```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
+                The DetNet Edge Node TSN function MUST support the TSN Stream
+                identification functions and the related managed objects as
+                defined in <xref target="IEEE8021CB">IEEE 802.1CB</xref> and
+                <xref target="IEEEP8021CBdb">IEEE P802.1CBdb</xref> to
+                recognize the App-flow related packets.  The service Proxy
+                presents TSN flows as a single application to a DetNet Flow.
+                The DetNet Flow is a statically defined service that connects
+                TSN service proxy capable Nodes. 
+                
+                <!-- This is interworking
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: No, this is not interworking!!! 
Interworking is when we reuse the seq.num of the TSN domain and copy it to the d-CW. Furthermore, we are placing e.g., replication points in TSN domain and elimination points in the DetNet domain.

Here text is about simple that edge nodes MUST be capable to recognize TSN streams and to map them to DetNet flows. 

My proposal is to revert to original text.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 4]
```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
-		<xref target="IEEE8021CB">IEEE 802.1CB</xref> and 
-		<xref target="IEEEP8021CBdb">IEEE P802.1CBdb</xref> to recognize 
-		the App-flow related packets. 
+          <t>
+                The DetNet Edge Node TSN function MUST support the TSN Stream
+                identification functions and the related managed objects as
+                defined in <xref target="IEEE8021CB">IEEE 802.1CB</xref> and
+                <xref target="IEEEP8021CBdb">IEEE P802.1CBdb</xref> to
+                recognize the App-flow related packets.  The service Proxy
+                presents TSN flows as a single application to a DetNet Flow.
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: TSN flow --> TSN stream (this is official IEEE terminology)

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 5]
```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
+                The DetNet Flow is a statically defined service that connects
+                TSN service proxy capable Nodes.
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: Delete new sentences here. What is "The DetNet Flow is a statically defined service" ??? We have definition for DetNet flow.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 6]
```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
+                TSN FRER functions service protection function interworking is
+                possible between the TSN and the DetNet domains. Such service
+                protection interworking scenarios MAY require to copy sequence
+                number fields from TSN (L2) to PW (MPLS) encapsulation.
+                However, such interworking is out-of-scope in this document and
+                left for further study.
+        </t>
+        <t>
+                The Service proxy SHALL support TSN MAC addresses allowing a
+                TSN connection to connect to another TSN network over the
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: TSN conenction = TSN stream ???
Sentence is unclear.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 7]
```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
+                DetNet Network. A service proxy SHALL be configured as either a
+                unicast DetNet application or a multicast application.  Service
+                proxies that support unicast service SHALL support TSN unicast
+                MAC addresses and Service proxies that support multicast
+                services SHALL support TSN multicast addresses.  The TSN
+                specific functions on the DetNet Edge Node SHOULD participate
+                in TSN Stream reservation protocols allowing TSN to establish
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: Reservation protocols we do not intend to cover here. Also, they are not used in fully centralized model. Let's discuss this whole paragraph, whether we need this new text.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 8]
```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
+                flows to the DetNet Edge node.  At the DetNet Edge node the TSN
+                functions are tunneled through the DetNet flow.
+        </t>
+        <t>
+                A static MPLS DetNet flow is configured to carry any number of
+                TSN flows. The Service proxy bandwidth profile SHOULD match the
+                bandwidth allocated to the MPLS Detnet Flow.  
+        </t>
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml:440]: What is a "static MPLS Detnet flow"? I think You mean here "MPLS DetNet flow".
I think the service proxy does not have a BW profile. Service proxy is a function. The DetNet service definition has a BW profile.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 9]
```diff
@@ -465,78 +499,103 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
 
    <t>
-	[Editor's note: This section covers management/control plane related
-	implications of creation, mapping, removal of DetNet flow IDs, their
-	related parameters and, when needed, the configuration of PREOF.]
+        TSN Stream(s) to DetNet flow mapping related information are
+        required only for the service proxy function of MPLS (DetNet) Edge nodes. 
+        From the Data Plane perspective TSN flows are application flows for 
+        DetNet. The proxy behaves as an Application interface for DetNet and 
+        as a TSN relay node for TSN Ethernet flows.
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: See comment above. Whole DetNet domain = TSN relay node, Proxy = port of TSN relay node.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 10]
```diff
@@ -465,78 +499,103 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
-	RSVP-TE and MSRP behaves differently. Therefore management and control
-	plane design is an important aspect of scenarios, where	mapping between 
-	DetNet and TSN is required.
+        The service Proxy is configured as a DetNet application and all functions are 
+        as described in <xref target="I-D.ietf-detnet-mpls"/>. As far as 
+        the TSN Network is concerned the service proxy is a TSN relay node that supports
+        one or more of TSN destination MAC addresses. These MAC address may be unicast or
+        multicast but the Detnet domain service proxy supports only one type at a time.
+        The service proxy is configured as a unicast or multicast Application for the DetNet 
+        interface.
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: "I-D.ietf-detnet-mpls" does not defines the proxy function. I have also concerns of such terminology "TSN destination MAC address". I think you mean here the "dst MAC address of the TSN stream".

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 11]
```diff
@@ -465,78 +499,103 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
+        Note that, as the DetNet network is just a portion of the end to end TSN
+        path (i.e., single hop from Ethernet perspective), some parameters
+        (e.g., delay) may differ significantly.  Since there is no interworking
+        function the bandwidth of DetNet network is assumed to be set large enough to
+        handle all TSN Flows it will support.  At the egress of the Detnet Domain the MPLS
+        headers are stripped and the TSN flow continues on as a normal TSN
+        flow. 
+</t>
+
+<!--    <<<<This is service interworking text>>>>
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: No, see comment above. This section is for Controller plane consideration. Mapping of TSN Streams to DetNet flows is not an interworking.

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml [hunk 12]
```diff
@@ -465,78 +499,103 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
...
    </t>
-
- </section> <!-- End of Management and Control Plane COnsiderations -->
+   <t>
+        Configuration of TSN specific functions (e.g., FRER)
+        inside the TSN network is a TSN domain specific decision 
+        and may not be visible in the DetNet domain. Service protection
+        interworking scenarios are left for further study.
+   </t>
+-->
```
Comment (vb-balazs) [inline tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml]: Remove

File: tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml (hunk 2 above)
Comment (dfedyk) [reply tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml:213]: OK as agreed per call

**Review by dfedyk — COMMENTED** [review]
OK as agreed.

**Review by dfedyk — COMMENTED** [review]
Seems I cannot approve. These changes are as agreed.

**Review by vb-balazs — APPROVED** [review]

**Pull request merged**